relay/paseo/build.rs

```diff
@@ -6,20 +6,25 @@
 // the Free Software Foundation, either version 3 of the License, or
 // (at your option) any later version.
 
-// Substrate is distributed in the hope that it will be useful,
+// Polkadot is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
 // You should have received a copy of the GNU General Public License
 // along with Paseo.  If not, see <http://www.gnu.org/licenses/>.
 
-use substrate_wasm_builder::WasmBuilder;
+#[cfg(all(feature = "std", not(feature = "metadata-hash")))]
+fn main() {
+	substrate_wasm_builder::WasmBuilder::build_using_defaults()
+}
 
+#[cfg(all(feature = "std", feature = "metadata-hash"))]
 fn main() {
-	WasmBuilder::new()
-		.with_current_project()
-		.import_memory()
-		.export_heap_base()
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash("DOT", 10)
```

Comment:
According to `system.properties` this should be `"PAS", 10`.
```suggestion
		.enable_metadata_hash("PAS", 10)
```

Reply:
thank you man!